### PR TITLE
fix: don't upgrade pandas

### PR DESCRIPTION
**Pull request by odinuv**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -34,7 +34,7 @@ RUN pip3 install --no-cache-dir --upgrade --force-reinstall \
         matplotlib \
         numpy \
         nose \
-        pandas \
+        pandas\<1.0\
```
Comment (pivnicek) [inline Dockerfile:37]: extra `\`?

**Review by pivnicek — COMMENTED** [review]
is there an extra slash here?

File: Dockerfile (hunk 1 above)
Comment (odinuv) [reply Dockerfile:37]: bash command

Comment (odinuv) [reply Dockerfile:37]: image

Comment (pivnicek) [reply Dockerfile:37]: ok!

**Review by pivnicek — APPROVED** [review]

**Pull request merged**